spec/ngsild_schema.json

```diff
@@ -0,0 +1,289 @@
+            ]
+        },
+
+
+        "EntityInfo": {
+            "type": "object",
+            "required": ["id"],
+            "properties": {
+                "id": {
+                    "type": "string"
+                },
+                "idPattern": {
+                    "type": "string"
+                },
+                "type": {
+                    "type": "string"
+                }
+            }
+        },
+
+        "EntityInfo_correct_definition_not_compilable_in_swagger_2_0": {
+            "anyOf": [
+                {
+                    "type": "object",
+                    "required": ["id"],
+                    "properties": {
+                        "id": {
+                            "type": "string"
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "required": ["idPattern"],
+                    "properties": {
+                        "idPattern": {
+                            "type": "string"
+                        }
+                    }
+                },
+                {
+                    "type": "object",
+                    "required": ["type"],
+                    "properties": {
+                        "type": {
+                            "type": "string"
+                        }
+                    }
+                }
+            ]
+        },
+
+
+        "RegistrationInfo": {
+            "type": "object",
+            "required": ["entities"],
+            "properties": {
+                "entities": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "$ref": "#/definitions/EntityInfo"
+                    }
+                },
+                "properties": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true
+                },
+                "relationships": {
+                    "type": "array",
+                    "minItems": 1,
+                    "items": {
+                        "type": "string"
+                    },
+                    "uniqueItems": true
+                }
+            }
+        },
+
+
+        "TimeInterval" : {
+            "type": "object",
+            "required": ["start"],
+            "properties": {
+                "start": {
+                    "type": "string",
+                    "format": "date-time"
+                },
+                "end": {
+                  "type": "string",
+                  "format": "date-time"
+                }
+            }
+        },
+
+
+        "GeoQuery" : {
+            "type": "object",
+            "required": ["geometry","coordinates","georel"],
+            "properties": {
+                "geometry": {
+                    "type": "string"
```

Comment:
we need to enumerate here the possible values from GeoJSON

Reply:
Pending